service/kvmconfig/v3/resource/clusterrolebinding/resource.go

```diff
@@ -0,0 +1,107 @@
+package clusterrolebinding
+
+import (
+	"reflect"
+
+	"github.com/giantswarm/microerror"
+	"github.com/giantswarm/micrologger"
+	"github.com/giantswarm/operatorkit/framework"
+	apiv1 "k8s.io/api/rbac/v1beta1"
+	"k8s.io/client-go/kubernetes"
+)
+
+const (
+	// Name is the identifier of the resource.
+	Name = "clusterrolebindingv3"
+)
+
+// Config represents the configuration used to create a new config map resource.
+type Config struct {
+	// Dependencies.
+	K8sClient kubernetes.Interface
+	Logger    micrologger.Logger
+}
+
+// DefaultConfig provides a default configuration to create a new config map
+// resource by best effort.
+func DefaultConfig() Config {
+	return Config{
+		// Dependencies.
+		K8sClient: nil,
+		Logger:    nil,
+	}
+}
```

Comment:
We agreed that we do not want to maintain the default config functions anymore. Sorry if this is not clearly enough communicated.

Reply:
ok